src/app/shared/components/template/services/instance/template-action.service.ts

```diff
@@ -37,7 +37,7 @@ export class TemplateActionService extends TemplateInstanceService {
   private templateTranslateService: TemplateTranslateService;
   private templateFieldService: TemplateFieldService;
 
-  constructor(public container: TemplateContainerComponent, injector: Injector) {
+  constructor(injector: Injector, public container?: TemplateContainerComponent) {
     super(injector);
```

Comment:
constructor orders swapped as we always need to pass an injector (which provides access to global services), but only sometimes a container. Various code snippets above and below to handle swap